docs/getting_started/configuration/prawini.rst

```diff
@@ -90,3 +90,41 @@ example, to use the settings defined for ``bot2`` as shown above, initialize
 
 A site can also be selected via a ``praw_site`` environment variable. This
 approach has precedence over the ``site_name`` parameter described above.
+
+Using Interpolation
+-------------------
+
+By default PRAW doesn't apply any interpolation on the config file but this can
+be changed with the ``config_interpolation`` parameter which can be set to
+"basic" or "extended".
+
+This can be useful to separate the components of the ``user_agent`` into
+individual variables, for example:
+
+.. _interpolation_site_example:
```

Comment:
Is this line needed? I think the code sample would work without it.

Reply:
you mean the `.. _interpolation_site_example:` part? I copied the example block from above but the pre_push script complained that the name was duplicated so I changed the name and retained the format, I think it creates an anchor in the document but I'm not sure.